src/index.js

```diff
@@ -90,12 +92,13 @@ function sendPayment (params) {
  * @param {String} [params.cancellationCondition] - Object, cancellation condition.
  *   If not provided, one will be generated.
  * @param {String} [params.caseId] = A notary case ID - if not provided, one will be generated
+ * @param {Boolean} [params.unsafeOptimisticTransport] - Set for optimistic mode
  * @param {String|Buffer} [params.ca] - Optional TLS CA if not using default CA (optional for https requests)
  */
 function executePayment (sourceTransfer, params) {
   return co(function * () {
     const isAtomic = !!params.notary
-    const isOptimistic = !params.receiptCondition && !isAtomic
+    const isOptimistic = !!params.unsafeOptimisticTransport
```

Comment:
It should throw an error if you don't sent `unsafeOptimisticTransport` and don't provide a condition